lib/browser/api/dialog.js

```diff
@@ -80,7 +80,8 @@ module.exports = {
       }
     }
 
-    let {buttonLabel, defaultPath, filters, properties, title} = options
+    let {buttonLabel, defaultPath, filters, properties, title,
+      message} = options
```

Comment:
This is fine to keep on a single line, we don't enforce hard line length limits in JS.